ompi/mca/pml/pml.h

```diff
@@ -523,6 +524,8 @@ struct mca_pml_base_module_1_0_1_t {
     uint32_t                              pml_max_contextid;
     int                                   pml_max_tag;
     int                                   pml_flags;
+
+    mca_pml_base_component_t             *pml_component;
```

Comment:
Isn't this equivalent to using mca_pml_base_selected_component or ompi_mtl_base_selected_component (which are supposed to either be NULL if the corresponding type was not selected or contain the component selected)?